scripts/set_final_strain_name.py

```diff
@@ -6,6 +6,11 @@ def replace_name_recursive(node, lookup):
     if node["name"] in lookup:
         node["name"] = lookup[node["name"]]
 
+    # drop the 'accession' which augur export automagically adds, as we use
+    # custom PPX_accession and insdcAccession fields, each with URLs attached
+    if 'accession' in node.get('node_attrs', {}):
+        del node['node_attrs']['accession']
+
```

Comment:
Is this still necessary if we just drop the `set_final_strain_name` script as per https://github.com/nextstrain/public/issues/5?

Reply:
If "accession" is in the metadata TSV (as it currently is) then `augur export v2` will always export it. Additionally it's being used as the node "name" as the current workflow (and parent PR, and current HEAD) use accession as the ID / newick-names.

PPX data has a well-formed strain name, so I'd consider using "strain" as the ID throughout both ingest & phylo, and thus having "PPX_accession" and "INSDC_accession"  but no "accession" key. There may be reasons to not want to do this that I am oblivious to.

If we did use a display-default of `"tip_label": "strain"` then we would still want to delete the automatically-added "accession" somehow, although it wouldn't be a deal-breaker.

Reply:
Finally, there's one more "modify-the-dataset-JSON" script which comes after the strain-name switch: `rule rename_and_ready_for_nextclade`. This script drops the "genome_clade_annotation" coloring and transfers those node_attrs (if present) to be branch labels. This can be avoided by:

1. Not exporting "genome_clade_annotation" as a coloring on the tree
2. Creating an intermediate node-data JSON which defines "genome_clade_annotation" as branch labels rather than node_attrs.

I'd do this but my test dataset (a/genome/all-time) doesn't have any "genome_clade_annotation" and searching for that string in the repo comes up empty.

Reply:
Looks like `genome_clade_annotation` used to be added in as "custom" clade labels:
 https://github.com/nextstrain/rsv/blob/7a2086af7ea2865a8caf5ad62a735969b3363771/workflow/snakemake_rules/clades.smk#L19-L20 

Since the workflow has switched to [using `augur clades` built-in options](https://github.com/nextstrain/rsv/blob/770fd86f148af309fc8c145867e79b79c8de6300/workflow/snakemake_rules/clades.smk#L18-L19) to define custom labels, I think it can be removed.

Reply:
> (I'm all for making changes here, but I don't want to do it as part of this PR.)

All for not doing as part of this PR, can continue discussion in https://github.com/nextstrain/rsv/issues/95